### DISCUSSION TITLE
Filtering Twitter or any other social media feeds

### DISCUSSION THREAD

**Discussion by omegasoft7** [WeAreMahsaAmini/Core#434] [Ideas]
Fake accounts are always trying to give wrong information in social media. This might make people:
-) less optimistic
-) nervous
-) getting cought because going to wrong place in a wrong time
And erc ...

The best way of fighting them is ro not seeing them.

It would be nice to have a filter to be able to see only posts that are from people, and not governmental army or governmental bots.

Maybe a:
-) web plugin that can filter those accounts
-) an app that people can easily share a profile with the app and report that profile.
-) and maybe an app that can show the filtered data of your Twitter or other social media accounts

**Comment by omegasoft7** [comment]
Even an app that can block reported accounts automatically would be a good solution for that

**Comment by pachol1401** [comment]
Please have a look on the comment here:
https://github.com/WeAreMahsaAmini/Core/discussions/256#discussioncomment-3833724
